docs/source/osds/scenarios.rst

```diff
@@ -1,6 +1,130 @@
 OSD Scenarios
 =============
 
+collocated
+----------
+This OSD scenario uses ``ceph-disk`` to create OSDs with collocated journals
+from raw devices.
+
+Use ``osd_scenario: collocated`` to enable this scenario. This scenario also
+has the following required configuration options:
+
+- ``devices``
+
+This scenario supports encrypting your OSDs by setting ``dmcrypt: True``.
```

Comment:
No, ``osd_scenario`` is just a string that represents a single osd scenario.  The ``dmcrypt`` option is used with ``osd_scenario``.  I'll add an example to make this more clear. It'll be something like:

```
osd_scenario: "collocated"
dmcrypt: true
```

This keeps us from having so many osd scenarios like ``collocated`` and ``collocated_dmcrypt``, etc.